getting-started/launch-demo.html.markerb

```diff
@@ -0,0 +1,172 @@
+
+Click **Confirm Settings** to confirm and deploy the demo app.
+
+Then return to your terminal to view the progress of the app deployment. For example:
+
+```output
+Waiting for launch data... Done
+Created app 'my-app-name' in organization 'personal'
+Admin URL: https://fly.io/apps/my-app-name
+Hostname: my-app-name.fly.dev
+Wrote config file fly.toml
+Validating /Users/username/my-app-dir/fly.toml
+Platform: machines
+✓ Configuration is valid
+==> Building image
+Searching for image 'flyio/hellofly:latest' remotely...
+image found: img_z1nr0lpjz9v5q98w
+
+Watch your deployment at https://fly.io/apps/my-app-name/monitoring
+
+Provisioning ips for my-app-name
+  Dedicated ipv6: 2a09:8280:1::42:a8f4
+  Shared ipv4: 66.241.124.213
+  Add a dedicated ipv4 with: fly ips allocate-v4
+
+This deployment will:
+ * create 2 "app" machines
+
+No machines in group app, launching a new machine
+Creating a second machine to increase service availability
+Finished launching new machines
+-------
+NOTE: The machines for [app] have services with 'auto_stop_machines = true' that will be stopped when idling
+-------
+Visit your newly deployed app at https://my-app-name.fly.dev/
+```
+
+## 4. Check your app's status
+
+After the app is deployed, use the `fly status` command to get the basic details about your new app.
+
+```cmd
+fly status
+```
+
+Example output:
+```cmd
+App
+  Name     = hellofly          
+  Owner    = personal       
+  Hostname = hellofly.fly.dev
+  Image    = flyio/hellofly:latest
+
+Machines
+PROCESS	ID            	VERSION	REGION	STATE  	ROLE   CHECKS	LAST UPDATED
+app    	148e453b7d7289	1      	ord   	started	      	        2023-05-17T17:39:37Z
+app    	5683d311c3228e	1      	ord   	stopped	      	        2023-05-17T17:38:44Z
+```
+
+In this example, the app has a DNS hostname of `hellofly.fly.dev`. Your app's name will, of course, be different. The app has two Machines running in the ord (Chicago) region. We recommend running at least two machines to improve availability in case of a single-host failure.
+
+## 5. Visit your app
+
+You can connect to your deployed app with the `fly apps open` command, which opens a browser on the http version of the site. That http connection will automatically be upgraded to an https secured connection (when using the `fly.dev` domain) to connect to it securely.
+
+You've successfully launched your first app on Fly.io!
+
+For fun, add `/<your-name>` to `fly apps open` and your name will be appended to the app's path to add an extra greeting from the hellofly application.
+
+```cmd
+fly apps open /fred
+```
+```out
+Opening http://hellofly.fly.dev/fred
+```
+
+## Deploy changes
+
+You probably don't need to make any changes to our demo app. But when you do make changes to an app on Fly.io, you'll always deploy it with:
```

Comment:
I'm being too literal as usual, but:

```suggestion
You probably don't need to make any changes to our demo app. But when you do make changes to an app on Fly.io, deploy a new release with:
```

(my reasoning is that we don't want folks to have to make any unnecessary mental shifts to understand that if you create or update a Machine with flyctl or the API, there's no `fly deploy` involved)